src/_includes/accessibility/accessibility-manual.html

```diff
@@ -93,19 +93,18 @@ <h4 class="margin-top-xl text-muted">If an icon represents an interactive elemen
 
   <div class="margin-bottom-lg margin-top-lg">
 {% highlight html %}
-<a href="#navigation-main" title="Skip to main navigation">
+<a href="#navigation-main" aria-label="Skip to main navigation">
   <i class="fa fa-bars" aria-hidden="true"></i>
 </a>
 {% endhighlight %}
     <small class="text-muted">an icon being used as a link to a navigation menu</small>
 
 {% highlight html %}
-<a class="btn btn-danger" href="path/to/settings" title="Delete">
-  <i class="fa fa-trash-o" aria-hidden="true"></i>
+<a class="btn btn-danger" href="path/to/settings" aria-label="Delete">
+  <i class="fa fa-trash-o" aria-hidden="true" title="Delete"></i>
```

Comment:
@adildostmohamed, thanks for the feedback. I wanted to leave one example present to show that a native tooltip (via `title` attr) can be added to this recommended syntax. I'll make that difference clear as well as leave a note about custom tooltips.